src/elements/header/header/header.visual.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { html } from 'lit';
```

Comment:
Could we also test the scrolling stuff?

Reply:
I gave it some try yesterday reducing the container height and using scrollTo/mockScrollTo, with no result :|

Reply:
I later try if I find a solution